arch/armv7a/jmp.S

```diff
@@ -6,90 +6,107 @@
  *
  * _setjmp, _longjmp, setjmp, sigsetjmp
  *
- * Copyright 2018 Phoenix Systems
- * Author: Kamil Amanowicz
+ * Copyright 2018, 2019 Phoenix Systems
+ * Author: Kamil Amanowicz, Aleksander Kaminski, Jakub Sarzynski
  *
  * This file is part of Phoenix-RTOS.
  *
  * %LICENSE%
  */
 
+ /* setjmp env buffer description
+  * Offset	Description
+  * 0		Signal mask flag - if != 0 then next value stores a signal mask
+  * 4		Signal mask
+  * 8		SP register
+  * 12-44	r4-r11 registers
+  * 44		LR register
+  * 48-80	d8-d15 registers
```

Comment:
```suggestion
  * 12-43	r4-r11 registers
  * 44		LR register
  * 48-79	d8-d15 registers
```

Reply:
Fixed, I also spotted tabs used for fancy alignment, changed to spaces